hackathon_site/event/static/event/styles/scss/styles.scss

```diff
@@ -52,107 +50,34 @@ strong {
 // End of general HTML stuff
 
 // Navbar stuff
-.logoNav100 {
-    height: 100%;
-    padding: 5px 0;
-}
-
 .logoNav {
-    height: 200px;
-    transition: height 200ms linear;
-    padding: 10px 0;
-
-    &Scrolled {
-        padding: 5px 0;
-        height: 100%;
-    }
-
-    &Side {
-        padding: 30px;
-        height: 150px;
-
-        a,
-        img {
-            height: 100% !important;
-            padding: 0 !important;
-
-            &:hover {
-                background-color: transparent !important;
-            }
-        }
-    }
-
-    @include responsive(lg-down) {
-        height: 64px;
-        padding: 5px 0;
-    }
-
-    @include responsive(md-down) {
-        height: 100%;
-    }
+    height: 100%;
+    padding: 5px;
 }
 
-.sidenav {
-    background: linear-gradient(180deg, #1a093e 30%, #2f1e52 60%, #4b3676 80%);
+.logoNavSide {
+    padding: 30px;
+    height: 150px;
 
-    li {
-        & > a {
-            color: #fff !important;
-        }
+    a,
+    img {
+        height: 100% !important;
+        padding: 0 !important;
 
-        & > .colorBtn {
-            color: #000 !important;
+        &:hover {
+            background-color: transparent !important;
         }
     }
-
-    li > a:hover,
-    nav ul a:hover {
-        background-color: rgba(0, 0, 0, 0.25);
-    }
 }
 
 .navbar {
     position: sticky;
     top: 0;
-    background-color: transparent;
-    box-shadow: none;
-    transition: background-color 200ms linear, top 200ms ease-in-out;
     z-index: 998;
-
-    &Scrolled {
-        background-color: rgba(39, 22, 72, 0.9);
-        box-shadow: 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12),
-            0 1px 5px 0 rgba(0, 0, 0, 0.2);
-    }
 }
 // End of navbar stuff
 
 // Landing section
-#landing {
-    position: relative;
-    z-index: 2;
-}
-
-.purpleLanding {
-    background-image: url("../../images/purple_landing.svg");
-    background-repeat: no-repeat;
-    background-size: auto 100%;
-    background-position: right top;
-    padding: 30px 0 50px;
-    margin-top: -64px;
-
-    @media only screen and (min-width: 2600px) {
-        background-position: left top;
-    }
-
-    @include responsive(sm-down) {
-        background-image: url("../../images/purple_landing_mobile.svg");
-        background-position: center bottom;
-        padding: 0 0 220px;
-        margin-top: -56px;
-    }
-}
-
```

Comment:
Can you revert everything above and including line 155, this is the styling that changes the width of the logo when you scroll, and styles the top of the page with the skyline